NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -137,7 +137,9 @@ def make_sim_shower(corsika, observer=None, detector=None, station_id=None):
         observer_position = np.zeros(3)
         observer_position[0], observer_position[1], observer_position[2] = -position[1] * units.cm, position[0] * units.cm, position[2] * units.cm
         observer_position = cs.transform_from_magnetic_to_geographic(observer_position)
-        sim_shower.set_parameter(shp.core, -observer_position + station_position)
+        core_position = (-observer_position + station_position)
```

Comment:
@cg-laser @fschlueter We need to do this for the Greenland simulations because the core positions are at 3km altitude. Is this also fine for Moores Bay and AERA simulations?

Reply:
Hi,
for AERA simulations that is fine (for the moment we call this function only with the "corsika" argument and set the core to the CoREAS/Corsika core). 
What you do here is setting the core altitude to 0 w.r.t sureface, right?

Reply:
this is just a x-y translation right? Because the station height is just 0. Looks good to me

Reply:
Yes, I just wasn't sure if anyone uses CoREAS simulations that have some other convention about this and wanted to make sure I don't break stuff for anyone else.